agr_java_core/src/main/java/org/alliancegenome/neo4j/entity/node/GeneticEntity.java

```diff
@@ -121,6 +121,7 @@ public GeneticEntity() {
 
 
     @JsonView({View.API.class, View.AlleleVariantSequenceConverterForES.class})
+    @JsonProperty(value = "crossReferences")
```

Comment:
This @Json field is going away with what @adamgibs is working on.

Reply:
without that JSON property the cross references will not get indexed

Reply:
it's always good to create new fields if needed instead of changing the existing because they might have been spawned at multiple places

Reply:
removing needs lot of testing

Reply:
yes I removed it and yes we are going to test it and Adam's PR is going to address all those issues. We need to have three methods for crossReferences not just one

Reply:
removed the JSON property.